### PR TITLE
Show LII for an AU member state

### PR DESCRIPTION
**Pull request by nickmwangemi**
### Member States listing page
image

### Member State detail page
image

### CODE REVIEW

File: africanlii/views/home.py [hunk 1]
```diff
@@ -39,92 +40,8 @@ def get_context_data(self, **kwargs):
...
-                "name": "ZimLII",
-                "country": "Zimbabwe",
-                "url": "https://zimlii.org",
-                "logo": "images/liis/zimlii.png",
-            },
-        ]
+        context["liis"] = show_liis(self.request)
+
         for lii in context["liis"]:
             lii["domain"] = lii["url"].split("/", 3)[2]
```
Comment (longhotsummer) [inline africanlii/views/home.py:46]: This is just to save us writing the domain a thousand times. You can safely move it to where you declare `LIIS` so it's only done once.

File: africanlii/context_processors.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    {
+        "name": "ZimLII",
+        "country": "Zimbabwe",
+        "url": "https://zimlii.org",
+        "logo": "images/liis/zimlii.png",
+    },
+]
+
+
+def show_liis(request):
```
Comment (longhotsummer) [inline africanlii/context_processors.py:89]: this isn't doing anything and is just weird. Just use `LIIS` in the places where this is called.

File: africanlii/templates/africanlii/_lii_info.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{% load i18n %}
+{% for lii in liis %}
+  {% if lii.country == member_state.country.name %}
+    <div>
+      {% blocktrans trimmed with lii_name=lii.name %}
+        Visit {{ lii_name }} for more legal information.</div>
```
Comment (longhotsummer) [inline africanlii/templates/africanlii/_lii_info.html:6]: the closing `</div>` is incorrect.

**Review by longhotsummer — DISMISSED** [review]

**Review by actlikewill — APPROVED** [review]
looks good

**Pull request merged**